nemo-wasm/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "nemo-wasm"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+js-sys = "0.3.63"
+nemo = { path = "../nemo", features = [ "js" ], default-features = false }
+nemo-physical = { path = "../nemo-physical", default-features = false }
+wasm-bindgen = "0.2"
+
+[profile.release]
+lto = true
```

Comment:
```
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
```

Reply:
Actually I am not sure, we want to enable lto globally on every release build. Maybe we should add a `release-lto` profile specifically for wasm?

Reply:
Might be a good idea. Having it below `nemo-wasm` will, however, just build the wasm without LTO, which we certainly also don't want to do.

Reply:
It seems there's a way to [override profiles for workspace members](https://doc.rust-lang.org/cargo/reference/profiles.html#overrides), that way we can have a `release` profile for `nemo-wasm` that has `lto`, while not needing to have it on, e.g., `nmo-cli`.

Reply:
Actually no:
```
error: failed to parse manifest at `/home/matze/Documents/Studium/6/knowsys/nemo/Cargo.toml`

Caused by:
  `lto` may not be specified in a `package` profile
```

Reply:
Yeah, we found that already and have agreed to take out the profile for now. Once `wasm-pack` can deal with custom profiles, we can re-enable this.

Reply:
In this case I would advocate for enabling `lto` globally.